app.js

```diff
@@ -88,8 +116,14 @@ function refreshCurrentGameEl() {
 
 function displayAllGames() {
     // clear out the past games list in the DOM
-
+    pastGamesEl.textContent = '';
     // loop through the past games in state
+    for (let game of pastGames){
+        const gameEl = renderGame(game.name1, game.name2, game.score1, game.score2);
+        pastGamesEl.append(gameEl);}
     // use the renderGame function to render and append a past game for each past game in state
+    renderGame();
     // again, review the renderGame function in render-utils.js. How many arguments does it take? What order does it take them in?
+    teamOneInput.value = '';
```

Comment:
You wrote a little event listener to do this input clear out for you. However, if you were putting these somewhere I'd probably put them inside the name submit event handler so when we submit our two names it clears them out there rather than when we call this `displayAllGames()` function